### DISCUSSION TITLE
Getting cp65001 in python after installing and enabling OhMyPosh

### DISCUSSION THREAD

**Discussion by russ960** [JanDeDobbeleer/oh-my-posh#1616] [Troubleshoot] [answered]
I installed via chocolaty OhMyPosh.  After I enabled it and added to my $PROFILE I started getting cp65001 error messages in python 2.7.  Now I don't really use python 2.7 for any coding but gsutil from Google does have a dependency on it and after recent reboot gsutil stopped working.  Once I commented out ohmyposh it started working correctly.  I'm not sure how to address this and I cannot find anyone with the same issue in some searches I performed.  Any help welcomed.

Python version: 2.7.14
Oh My Posh version: 6.42.0

Enablement line from $PROFILE:
`oh-my-posh --init --shell pwsh --config ~/jandedobbeleer.omp.json | Invoke-Expression`

**Comment by JanDeDobbeleer** [comment, marked as answer]
You need to file a bug at gsutil (or Python) as it doesn't seem to handle UTF8 right. Without UTF8, the characters of oh-my-posh can't render correctly (that's how the charset works).

**Comment by russ960** [comment]
I wanted to add the comment here after the above suggestion I found that Google has finally removed this dependency on 2.7 so if using gsutil 5.0 or later removes 2.7 dependency.
